src/components/Autosuggest/Autosuggest.scss

```diff
@@ -9,7 +9,7 @@
         transition-property: box-shadow, transform, border-color;
         transition-duration: var(--transition-duration);
         border-radius: var(--border-radius);
-        z-index: 10;
```

Comment:
this is actually an overlay, the dropdown of autosuggest, isn't it? Then it should be in the second range.

Reply:
since we changed the range, this should go even higher :-|